index.html

```diff
@@ -23,13 +24,40 @@
         <div class="mode">
             <p class="mode__type">MODE: Who is this character?</p>
             <div class="mode__rules">
-                <p class="mode__title"><i class="fas fa-graduation-cap"></i> Mode rules</p>
+                <p class="mode__title"><i class="fas fa-graduation-cap"></i><span> Mode rules</span></p>
                 <p class="mode___description">You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select who from Star Wars is showed on the left (Jar Jar Binks right now) from available options.</p>
             </div>
+            <div class="mode__rules mode__hall">
+                <p class="mode__title"><i class="fas fa-id-badge"></i><span> Mode ranking</span></p>
+                <div class="ranking__table">
+                    <div class="ranking__table--header">
+                        <p class="ranking__header--bold">Place</p>
+                        <p class="ranking__header--bold">Player</p>
+                        <p class="ranking__header--bold">Answered</p>
+                    </div>
+                    <div class="ranking__table--body">
+                        <div class="ranking__table--row">
+                            <p class="ranking__place">1st</p>
+                            <p class="ranking__name">Ania</p>
+                            <p class="ranking__score">15/20</p>
+                        </div>
+                        <div class="ranking__table--row">
+                            <p class="ranking__place">2nd</p>
+                            <p class="ranking__name">Mateusz</p>
+                            <p class="ranking__score">14/30</p>
+                        </div>
+                        <div class="ranking__table--row">
+                            <p class="ranking__place">3rd</p>
+                            <p class="ranking__name">Leia Organa</p>
+                            <p class="ranking__score">1/23</p>
+                        </div>
+                    </div>  
+                </div>                  
+            </div>
         </div>        
     </div>
     <div class="buttons">
-        <button class="button button--ranking"><i class="fas fa-id-badge"></i> Hall of fame</button>
+        <button class="button button--ranking"><i class="fas fa-id-badge"></i> <span>Hall of fame</span></button>
```

Comment:
ten span nie jest potrzebny

Reply:
tak jak pisałam, bez spanów "zjadało" mi ikonę. potestuję może inne metody wyciągania tej wartości przez DOM, żeby nie było spana

Reply:
a fakt, nie zauważyłem. to można dać: buttonRanking.innerHTML = '<i class="fas fa-graduation-cap"></i> Rules'; i nie trzeba ustawiac classList na zadnym elemencie. ale nie wiem czy tak się powinno robić, ja w tym nic złego nie widzę, nie jest to brane z inputa od użytkownika więc innerHTML chyba tutaj spoko. @ktprezes @asiaziola opinie?

Reply:
nie widzę w tym nic złego, jeżeli działa

Reply:
jedyne, co na razie działa, to jak dodaję klasę do spana przed tekstem przycisku i wtedy w js mogę operować querySelector+klasa bez posługiwania się dziećmi